scripts/get-flattened-publication.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -ueo pipefail
+
+USAGE="$0 HOST STORAGE_KEY
+Returns flat JSON corresponding with chain storage data returned by an RPC node.
+
+  HOST is a URL prefix against which an abci_query HTTP request may be issued,
+       e.g. \"https://xnet.rpc.agoric.net/\".
+  STORAGE_KEY is a chain storage key name, e.g. \"published.amm.pool0.metrics\".
+";
+
+URL_PREFIX="${1:-}";
+STORAGE_KEY="${2:-}";
+
+if [ ":${1:-}" = '--help' -o ":$URL_PREFIX" = : -o ":$STORAGE_KEY" = : -o $# -gt 2 ]; then
+  printf '%s' "$USAGE"
+  exit 64
+fi
+
+# Verify jq version.
+jq --version | awk '
```

Comment:
I'm more familiar with version checking by grep and having the condition in shell code. nbd.